### ISSUE TITLE
[Engine init] Exclusive full screen exception

### ISSUE THREAD

**Issue by TheOrneryNerd** [elnormous/ouzel#40]
if I set exclusive full screen setting to true in the Settings.ini file and start a new instance of the engine in Windows, the engine crashes in Window->init, as the window is initialized before the renderer.

I noticed the problem happens here:
File: NativeWindowWin.cpp
Method: switchFullscreen

Inside:
if (exclusiveFullscreen)
{
  //uninitialized D3D renderer is called
  // from the NativeWindowWin.
}

Am I doing anything wrong or is there a fix?

Thanks in advance

**Comment by elnormous** [comment]
No, you are doing everything right. Exclusive fullscreen is not yet fully supported, I am working on it. Thank you!
